src/libbson/doc/bson_oid_compare_unsafe.rst

```diff
@@ -0,0 +1,17 @@
+:man_page: bson_oid_compare_unsafe
+
+bson_oid_compare_unsafe()
+=========================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  static inline int
+  bson_oid_compare_unsafe (const bson_oid_t *oid1, const bson_oid_t *oid2);
+Description
```

Comment:
```diff
    bson_oid_compare_unsafe (const bson_oid_t *oid1, const bson_oid_t *oid2);
+ 
  Description
```

Add newline after code block and before Description to address the following Doxygen warning:

```
WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

Also applies to other instances in this PR.